Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -419,6 +454,31 @@ export class Kudu {
         }
     }
 
+    private async _getDeploymentDetailsFromPollURL(pollURL: string):Promise<any> {
+        var httpRequest = new webClient.WebRequest();
+        httpRequest.method = 'GET';
+        httpRequest.uri = pollURL;
+        var timeOutInMinutes
+        while(true) {
```

Comment:
Should we be careful and retry only for a finite amount of time ?

Reply:
We cant be sure about the time taken to deploy the package (similar to MSDeploy).